src/components/Main.js

```diff
@@ -2,67 +2,89 @@ import React from 'react';
 import axios from 'axios';
 import CityForm from './CityForm';
 import CityCard from './CityCard';
+import WeatherCard from './WeatherCard';
 
 
 class Main extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            searchQuery: '',
-            location: {},
-            error: false,
-            errorMessage: '',
-            cityMap: '',
-        }
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      location: {},
+      error: false,
+      errorMessage: '',
+      cityMap: '',
+      weatherArr: [],
+
+    };
+  }
+  handleInput = (e) => {
+    console.log(e.target);
+    // e.preventDefault();
+    this.setState({ searchQuery: e.target.value });
+    console.log(this.state.searchQuery);
+  };
+
+  // async await
+  handleSearch = async (e) => {
+    // code runs in the try block
+    e.preventDefault();
+    try {
+      const API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATION_IQ_KEY}&q=${this.state.searchQuery}&format=json`;
+      const res = await axios.get(API);
+      // you want res.data
```

Comment:
you can unpack the data object like `let { lat, lon } = res.data[0]`, if you want